_posts/2017-10-25-rails-girls-berlin-is-growing-up.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Rails Girls Berlin is growing up"
+date: "2017-10-25"
+coverImage: "IMG_0876b.jpg"
+---
+
+Rails Girls Berlin has existed since its first workshop in 2012. So far, we’ve organized more than 30 workshops with more than 1200 learners. From within our community, great projects such as [speakerinnen.org](http://speakerinnen.org), [rorganize.it](http://rorganize.it) and [diversitytickets.org](http://diversitytickets.org) have developed. Additionally, projects like [Rails Girls Summer of Code](https://railsgirlssummerofcode.org/), [ClojureBridge Berlin](http://clojurebridge-berlin.org) and [Heart of Code](http://heartofcode.org) are partially built by people who found or refreshed their love for programming with us.
```

Comment:
`exists since` or `has been around since` and `1.200 learners` (easier to read imo)